src/databricks/labs/ucx/source_code/known.json

```diff
@@ -32580,6 +32591,71 @@
     "functorch": [],
     "torch": []
   },
+  "torchvision": {
+    "torchvision": [],
+    "torchvision._internally_replaced_utils": [],
+    "torchvision._meta_registrations": [],
+    "torchvision._utils": [],
+    "torchvision.datasets": [],
```

Comment:
Had to keep torchvision.datasets as one of the libraries has to be excluded, so just included all but torchvision.datasets.widerface
`    "torchvision.datasets.widerface": [
      {
        "code": "direct-filesystem-access",
        "message": "The use of direct filesystem references is deprecated: /"
      }
    ],`